src/pages/projects/LatestProjects.tsx

```diff
@@ -0,0 +1,100 @@
+import { t, Trans } from '@lingui/macro'
+import Grid from 'components/Grid'
+import Loading from 'components/Loading'
+import ProjectCard, { ProjectCardProject } from 'components/ProjectCard'
+import { ThemeContext } from 'contexts/themeContext'
+import { useInfiniteProjectsQuery } from 'hooks/Projects'
+import { useContext, useEffect, useRef } from 'react'
+
+export default function LatestProjects() {
+  const pageSize = 20
+  const {
+    theme: { colors },
+  } = useContext(ThemeContext)
+  const {
+    data: pages,
+    isLoading,
+    isFetchingNextPage,
+    hasNextPage,
+    fetchNextPage,
+  } = useInfiniteProjectsQuery({
+    orderBy: 'createdAt',
+    pageSize,
+    orderDirection: 'desc',
+  })
+  const loadMoreContainerRef = useRef<HTMLDivElement>(null)
+
+  const concatenatedPages = pages?.pages?.reduce(
+    (prev, group) => [...prev, ...group],
+    [],
+  )
+
+  // When we scroll within 200px of our loadMoreContainerRef, fetch the next page.
+  useEffect(() => {
+    if (loadMoreContainerRef.current) {
```

Comment:
Do you think we could make a shared function somewhere (maybe just in `index`) to avoid repeating lines `34:47` in two places?

Reply:
agreed with al, i've created a custom hook `useLoadMoreContent` so it can serve us in the future.